docs/concepts/executor/serve.md

```diff
@@ -236,6 +277,12 @@ print(client.post(on='/', inputs=Document(), stream=False).texts)
 ['hello']
 ```
 
+````{admonition} Hint
+:class: hint
+You can also export an Executor deployment to kubernetes YAML files using the CLI commad, in case you define a Deployment YAML config:
```

Comment:
```suggestion
You can also export an Executor deployment to kubernetes YAML files using the CLI command, in case you define a Deployment YAML config:
```